pkg/guestconversion/guestconversion.go

```diff
@@ -20,15 +20,15 @@ var (
 // MakeGuestConversionJobSpec creates a job spec for a virt-v2v job,
 // containing a volume and a mount for each volume on the VM, as well
 // as a volume and mount for the config map containing the libvirt domain XML.
-func MakeGuestConversionJobSpec(vmSpec *v1.VirtualMachine, libvirtConfigMap *corev1.ConfigMap) *batchv1.Job {
+func MakeGuestConversionJobSpec(vmSpec *v1.VirtualMachine, libvirtConfigMap *corev1.ConfigMap, privilegedSA string) *batchv1.Job {
```

Comment:
The privilegedSA is a concern of the GuestConversionJob.
Can it be obtained within guestconversion.MakeGuestConversionJobSpec instead of passing it from the provider ?
It will required passing the k8s client to this method.